src/overlays/actors/ovl_En_Poh/z_en_poh.h

```diff
@@ -2,18 +2,19 @@
 #define Z_EN_POH_H
 
 #include "global.h"
+#include "objects/object_po/object_po.h"
 
 struct EnPoh;
 
 typedef void (*EnPohActionFunc)(struct EnPoh*, PlayState*);
 
 typedef enum EnPohBodyPart {
-    /*  0 */ ENPOH_BODYPART_0,
-    /*  1 */ ENPOH_BODYPART_1,
-    /*  2 */ ENPOH_BODYPART_2,
-    /*  3 */ ENPOH_BODYPART_3,
-    /*  4 */ ENPOH_BODYPART_4,
-    /*  5 */ ENPOH_BODYPART_5,
+    /*  0 */ POE_BODYPART_LEFT_FOREARM,
+    /*  1 */ POE_BODYPART_LEFT_UPPER_ARM,
+    /*  2 */ POE_BODYPART_RIGHT_ARM_HAND,
+    /*  3 */ POE_BODYPART_RIGHT_FOREARM,
+    /*  4 */ POE_BODYPART_BOTTOM_CLOAK,
+    /*  5 */ POE_BODYPART_TOP_CLOAK,
     /*  6 */ ENPOH_BODYPART_6,
```

Comment:
```suggestion
    /*  5 */ POE_BODYPART_TOP_CLOAK,
    /*  6 */ POE_BODYPART_6,
```
Just one more comment, since you're changing the prefix, could you apply that to the other body parts too